### PR TITLE
Add a README for llama subtitle sumerizer pipeline example

### PR DESCRIPTION
**Pull request by yeonykim2**
- Add a README file to guide users to build/run the pipeline example.

### CODE REVIEW

File: Tizen.native/llama_68M.cpp/README.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+  $ sdb push models/llama-68m-chat-v1.fp16.gguf /usr/lib/nnstreamer/bin/models/
+  $ sdb push subtitles.srt /usr/lib/nnstreamer/bin/
+  $ sdb push big_buck_bunny_trailer_480p.webm /usr/lib/nnstreamer/bin/
+  ```
+
+
+
+## Run pipeline in sdb shell
+
+Following gst-launch example make llama.cpp do summerize on the sample `subtitles.srt` with llama-68m-chat-v1.fp16.gguf model.
```
Comment (wooksong) [inline Tizen.native/llama_68M.cpp/README.md:49]: The following GST-launch example makes a ```llama.cpp``` summarize on the sample subtitle, ```subtitles.srt```, with the ```llama-68m-chat-v1.fp16.gguf``` model.

**Review by anyj0527 — APPROVED** [review]
Good start for LLM with NNStreamer/Tizen, Cool👍
Someday more large/powerful LLM model usages could be feasible.

File: Tizen.native/llama_68M.cpp/README.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+- Build nnstreamer-llama-cpp rpm package
+  ```bash
+  $ git clone https://github.com/yeonykim2/llama.cpp && cd llama.cpp && git checkout nnstreamer_llama_subtitle_summarizer
+  $ gbs build -A aarch64
+
+  # check contents of the RPM file
+  $ ls ~/GBS-ROOT/local/repos/tizen/aarch64/RPMS
+  > nnstreamer-llama-68m-gguf-1.0.0-0.aarch64.rpm ...
+  $ cp ~/GBS-ROOT/local/repos/tizen/aarch64/RPMS/nnstreamer-llama-68m-gguf-1.0.0-0.aarch64.rpm .
+  $ rpm2cpio nnstreamer-llama-68m-gguf-1.0.0-0.aarch64.rpm | cpio -idmv
```
Comment (niley7464) [inline Tizen.native/llama_68M.cpp/README.md:26]: If this command is just for checking contents, and not for extracting the compressed files, how about using this options?
```sh
$ rpm -qlp nnstreamer-core-2.4.1-0.x86_64.rpm 
/usr/lib/nnstreamer/decoders/libnnstreamer_decoder_bounding_boxes.so
/usr/lib/nnstreamer/decoders/libnnstreamer_decoder_direct_video.so
/usr/lib/nnstreamer/decoders/libnnstreamer_decoder_image_labeling.so
/usr/lib/nnstreamer/decoders/libnnstreamer_decoder_image_segment.so
/usr/lib/nnstreamer/decoders/libnnstreamer_decoder_octet_stream.so
/usr/lib/nnstreamer/decoders/libnnstreamer_decoder_pose_estimation.so
/usr/lib/nnstreamer/decoders/libnnstreamer_decoder_tensor_region.so
/usr/lib/nnstreamer/filters/libnnstreamer_filter_cpp.so
/usr/lib64/gstreamer-1.0/libgstedge.so
/usr/lib64/gstreamer-1.0/libnnstreamer.so
/usr/lib64/libnnstreamer.so
/usr/share/licenses/nnstreamer-core
/usr/share/licenses/nnstreamer-core/LICENSE
```
https://linux.die.net/man/8/rpm

Comment (yeonykim2) [reply Tizen.native/llama_68M.cpp/README.md:35]: thanks, i updated it.

**Review by niley7464 — APPROVED** [review]
LGTM!

**Review by myungjoo — APPROVED** [review]

**Pull request merged**